timeless/sms.py

```diff
@@ -29,26 +53,21 @@ def __init__(
 
     def make_base_payload(self):
         """Make base payload with basic data for provider
-        @todo #46:30min setup rultor settings to install python 3.6 (3.6.7
-         if is possible) according to project software environment.
-         After that we can use f-strings in python
```

Comment:
@vladarefiev why do you remove this puzzle in this PR? It has nothing to do with the task.

Reply:
@amihaiemil oopls, i forgot remove it in my last PR %)
you are very attentive, but you were a reviewer there - https://github.com/timelesslounge/timelessis/pull/135